metadata-ingestion/setup.py

```diff
@@ -398,7 +398,7 @@
     "unity-catalog": databricks | sql_common | sqllineage_lib,
     # databricks is alias for unity-catalog and needs to be kept in sync
     "databricks": databricks | sql_common | sqllineage_lib,
-    "fivetran": snowflake_common | bigquery_common,
+    "fivetran": snowflake_common | bigquery_common | {"sqlalchemy-bigquery>=1.4.1"},
```

Comment:
maybe just move this into bigquery_common

Reply:
Done